README.md

```diff
@@ -18,21 +18,45 @@ After installing the package you can import the tracking component into your cod
 
 ### Client-side integration
 
-On the client-side this package can be used to configure [o-tracking], setup click tracking, and send a page view event.
+**Configure [o-tracking]**
+
+This package can be used on the client-side. Initialise the component with [configuration options](#options).
 
 ```js
 import * as nTracking from '@financial-times/n-tracking';
+const options = {}
+const oTracking = nTracking.init(options); 
+```
+ 
+**Send custom events**
 
-const oTracking = nTracking.init(options);
-
+```js
+import * as nTracking from '@financial-times/n-tracking';
+const context = { customData: 12345 }
 nTracking.broadcast('oTracking.event', {
   category: 'page',
   action: 'custom-event',
+  ...context
 });
 ```
 
-To initialise the component you'll need to provide it with several [configuration options](#options).
+**Set up click tracking**
```

Comment:
Click tracking is setup by default when initialising

Reply:
👍 Cheers